class CustomConflictError(Exception):
    pass


class CustomBadRequestError(Exception):
    pass
